File: frontend/lib/models/nutrition.dart
class Nutrition {
  final int calories;
  final int proteins;
  final int carbs;
  final int fats;

  Nutrition(
      {required this.calories,
      required this.proteins,
      required this.carbs,
      required this.fats});

  factory Nutrition.fromJson(dynamic json) {
    return Nutrition(
        calories: json['calories'] as int,
        proteins: json['proteins'] as int,
        carbs: json['carbs'] as int,
        fats: json['fats'] as int);
  }

  @override
  String toString() {
    return 'Recipe {calories: $calories, proteins: $proteins, carbs: $carbs, fats: $fats}';
  }
}

File: backend/src/main/java/com/brainfood/models/Nutrition.java
package com.brainfood.models;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Nutrition {

    private int calories;

    private int proteins;

    private int carbs;

    private int fats;

}
